systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/PodUtils.java

```diff
@@ -265,23 +265,20 @@ public static void waitUntilPodsStability(List<Pod> pods) {
 
     /**
      * Log actual pod statuses list by prefix name
-     * @param kind - custom resource / deployment kind - Kafka, KafkaBridge etc.
      * @param name - custom resource / deployment name - used for prefix
      * @param log - ArrayList - add statuses, pods and conditions for future display
      */
-    public static void logCurrentPodStatus(String kind, String name, List<String> log) {
-        if (!(kind.equals("KafkaConnector"))) {
-            log.add("\nPods with conditions and messages:\n\n");
-            for (Pod pod : kubeClient().listPodsByPrefixInName(name)) {
-                log.add(pod.getMetadata().getName() + ":");
-                for (PodCondition podCondition : pod.getStatus().getConditions()) {
-                    if (podCondition.getMessage() != null) {
-                        log.add("\n\tType: " + podCondition.getType() + "\n");
-                        log.add("\tMessage: " + podCondition.getMessage() + "\n");
-                    }
+    public static void logCurrentPodStatus(String name, List<String> log) {
+        log.add("\nPods with conditions and messages:\n\n");
```

Comment:
Maybe we should use different name? Or log it directly in that function? Now it's a little bit confusing

Reply:
I think we can add it directly to log function, I think that we will never use without second method.